gematria/datasets/python/BUILD.bazel

```diff
@@ -94,7 +94,6 @@ gematria_py_test(
         "//gematria/proto:basic_block_py_pb2",
         "//gematria/proto:canonicalized_instruction_py_pb2",
         "//gematria/proto:throughput_py_pb2",
-        "@rules_python//python/runfiles",
```

Comment:
Is there anything we can drop from the `WORKSPACE` file?

Reply:
Yeah, looks like we can just remove the entire `rules_python` repo. I'll do a follow up patch to pull that out.